### DISCUSSION TITLE
Adding another argument to stop scanning once first  jar file found

### DISCUSSION THREAD

**Discussion by tjyang** [fox-it/log4j-finder#68] [Ideas]
From using batch scanning toward thousands system controlled by saltstack server.
I found this feature is important, so that when a system has huge file system and contain many .jar files, log4j-finder to terminate the scan quickly without timeout.  We can run another scan afterword if full list is needed.

**Comment by yunzheng** [comment]
Do you mean stop scanning when the first VULNERABLE hit is found?

**Comment by tjyang** [comment]
yes, stop the scanning to save time when -q (quick scan) is provided ;)

**Comment by yunzheng** [comment]
`-q` is already used for `--quiet` 😄
Maybe `-Q/--quick-scan` or `-s/--stop-on-first-hit`.

But maybe even better if it's configurable, something like `--alert-threshold=1` or `--hit-threshold=1`
